### PR TITLE
City-office: allow different order of values in `address` and `address_map` types

### PR DESCRIPTION
**Pull request by angelikatyborska**
Closes https://github.com/exercism/elixir/issues/1124

### CODE REVIEW

File: exercises/concept/city-office/test/form_test.exs [hunk 1]
```diff
@@ -85,15 +85,25 @@ defmodule FormTest do
           declaration == :type && type_name == unquote(type_name)
         end)
 
+      assert type,
+             "expected the module #{unquote(module_name)} to have a public type named #{unquote(type_name)}"
+
```
Comment (angelikatyborska) [inline exercises/concept/city-office/test/form_test.exs:90]: This is an unrelated bug fix. This assertion was too late, causing a cryptic `no match of right hand side value: nil` error when a type was missing.

**Review by jiegillet — APPROVED** [review]
Looks good, thank you for closing the issue!
